File: 1. Beginner/3145/3145.cs
using System; 
class URI {
    static void Main(string[] args) { 
        int n, x;
        double r;

        string[] inputs = Console.ReadLine().Trim().Split(' ');
        n = Int32.Parse(inputs[0]);
        x = Int32.Parse(inputs[1]);
        
        r = (double) x / (n + 2);

        Console.WriteLine(r.ToString("F2"));
    }
}

File: 1. Beginner/3145/3145.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input).unwrap();
    let nums: Vec<f64> = input.trim().split_whitespace()
                               .map(|s| s.parse().unwrap())
                               .collect();
    let n = nums[0];
    let x = nums[1];

    println!("{:.2}", x / (n + 2.0));
}